comp/haagent/impl/haagent_testutils_test.go

```diff
@@ -9,25 +9,43 @@ import (
 	"testing"
 
 	"github.com/DataDog/datadog-agent/comp/core/config"
+	configComponent "github.com/DataDog/datadog-agent/comp/core/config"
```

Comment:
this is already imported in the line above

Reply:
@NouhaManai96 I finally split the PR into two, this one only contains `config_id` metadata